src/pybind/mgr/dashboard/frontend/e2e/block/images.e2e-spec.ts

```diff
@@ -16,4 +16,24 @@ describe('Images page', () => {
     page.navigateTo();
     expect(Helper.getBreadcrumbText()).toEqual('Images');
   });
+
+  it('should show three tabs', () => {
+    page.navigateTo();
+    expect(Helper.getTabsCount()).toEqual(3);
```

Comment:
I think we could put all 4 `expects` in the 1st `it`.
Don't see a reason to add overhead when none of the tests change state and can be tested in the same page.

Same applies to the mirroring tests.

Reply:
Should the description include all of these attributes as well? e.g.

`it('should show breadcrumb text and three tabs - images, trash, and overall performance', () => {`

Reply:
What should the description look like? Something more generic?

Reply:
I'm OK with leaving it like it is: `'should show three tabs'`

Reply:
Will make the requested changes - we do have existing tab tests in this format (see PRs #26151 and #25572) so we may want to consider altering those tests to the new format for consistencies sake.

Reply:
IMHO it would be nice to differ between the test which checks the number of displayed tabs and the test which actually checks the label of the tab - so my suggestion would be to have at least one ``it`` for the tab count and another ``it`` for checking the correct labels of the displayed tabs.

Reply:
@LenzGr I think it may make sense to do so in a separate PR for documenting purposes @callithea I like that idea as well

I am fine with either courses of action; whatever the team would like to do. I am OOO until Friday so I will implement whatever is decided on then :)

Reply:
Just talked with @callithea and the solution we reach is to add a `describe` that includes all the tests and a `beforeAll` that includes the `navigateTo`.
That way we only open the page once and can do all the tests we want.

```
  describe('Test main page', () => {
    beforeAll(() => {
      page.navigateTo();
    });

    it('should show three tabs', () => {
      expect(Helper.getTabsCount()).toEqual(3);
    });

    it('should show text for all tabs', () => {
      expect(Helper.getTabText(0)).toEqual('Images');
      expect(Helper.getTabText(1)).toEqual('Trash');
      expect(Helper.getTabText(2)).toEqual('Overall Performance');
    });
  });
```

Reply:
Should the breadcrumb test be included in this new sub-describe function, or just the tab tests?

Also, how do we want to go about retroactively changing the existing tests? I think a separate PR may be best so we can do them all in one swoop and simplify the documenting.

Reply:
Yes, please include that test as well.
We can do it in a separate PR.